source/package-manager/configs.js

```diff
@@ -52,3 +52,18 @@ export const yarnBerryConfig = {
 	throwOnExternalRegistry: true,
 	lockfiles: ['yarn.lock'],
 };
+
+/** @type {import('./types.d.ts').PackageManagerConfig} */
+export const bunConfig = {
+	cli: 'bun',
+	id: 'bun',
+	installCommand: ['bun', ['install', '--frozen-lockfile']],
+	installCommandNoLockfile: ['bun', ['install', '--no-save']],
+	versionCommand: version => ['npm', ['version', version]],
+	// Bun doesn't support publishing, so we use npm instead
+	publishCommand: arguments_ => ['npm', arguments_],
+	// TODO: Bun doesn't support config get registry, this should be added in the future
```

Comment:
Can you link to a Bun issue for this in the comment?

Reply:
Sure.